dotnet-dependency-injection/DependencyInjectionLifetimeScopes/Test/Controllers/ValuesControllerTest.cs

```diff
@@ -0,0 +1,25 @@
+using DependencyInjectionLifetimeScopes;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace DependencyInjectionLifetimeScopesTest
+{
+    public class ValuesControllerTest : IClassFixture<CustomWebApplicationFactory<Startup>>
```

Comment:
As discussed in the review, you could have used this opportunity to test different outcomes with different scopes. Just create. You didn't even need to use API or endpoints to demonstrate that. Maybe we can try experimenting this time?

Reply:
I believe this is not required anymore as we agreed to switch back to API endpoint based approach